src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:33235/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:33235/project/issues?id=sample&branch=feature4&issues=AWG8shS9o4O85XprFLfg&open=AWG8shS9o4O85XprFLfg)

[ISSUE_KEY]: AWG8shS9o4O85XprFLfg